astropy/nddata/bitmask.py

```diff
@@ -92,11 +276,15 @@ def interpret_bit_flags(bit_flags, flip_bits=None):
 
     Examples
     --------
-        >>> from astropy.nddata.bitmask import interpret_bit_flags
+
+        >>> from astropy.nddata.bitmask import interpret_bit_flags, extend_bit_flag_map
+        >>> ST_DQ = extend_bit_flag_map('ST_DQ', CR=1, CLOUDY=4, RAINY=8, HOT=16, DEAD=32)
         >>> "{0:016b}".format(0xFFFF & interpret_bit_flags(28))
         '0000000000011100'
         >>> "{0:016b}".format(0xFFFF & interpret_bit_flags('4,8,16'))
         '0000000000011100'
+        >>> "{0:016b}".format(0xFFFF & interpret_bit_flags('CLOUDY,RAINY,HOT', flag_name_map=ST_DQ))
+        '0000000000011100'
         >>> "{0:016b}".format(0xFFFF & interpret_bit_flags('~4,8,16'))
         '1111111111100011'
         >>> "{0:016b}".format(0xFFFF & interpret_bit_flags('~(4+8+16)'))
```

Comment:
Can you do this `'~(4+8+16)'` with named bits too?

Reply:
@pllim Yes, one can. It is identical: `name<->value`

Reply:
Maybe add an example?